scripts/dependency_install_functions.sh

```diff
@@ -80,13 +120,32 @@ install_protobuf() {
     else
       "$SUDO_CMD" cmake --build build --target install
     fi
+    # Build Shared library 
+    # Don't build shared, it messes up the static library linking because the
+    # cmake file installed are not compatible
+    # WARNING - static library will break if build with shared options on
+    #cmake -S . -B build \
+    #  -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+    #  -DBUILD_SHARED_LIBS=ON \
+    #  -Dprotobuf_BUILD_TESTS=OFF \
+    #  -DABSL_PROPAGATE_CXX_STD=ON \
+    #  -DCMAKE_INSTALL_PREFIX="${DATAFED_DEPENDENCIES_INSTALL_PATH}"
+    #cmake --build build -j 8
+    #if [ -w "${DATAFED_DEPENDENCIES_INSTALL_PATH}" ]; then
+    #  cmake --build build --target install
+    #else
+    #  "$SUDO_CMD" cmake --build build --target install
+    #fi
```

Comment:
Comments

Reply:
Good catch this is one of the examples of the code where you cannot install both the shared and static to the same location or make sees red and just grabs the shared libraries.